CONFIG/config.yml

```diff
@@ -0,0 +1,54 @@
+network:
+  subnet:
+    first_sheet:
+      c_green_a: [0, 1, 2, 3, 4, 13, 14, 15, 16, 17, 
+            18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 
+            28, 29, 91, 92, 93, 94, 96, 97,
+            98, 99, 100, 101, 102, 103, 104, 108]
+      c_grey_a: [42, 43, 120, 121]
+      c_yellow_a: [11, 12, 89, 90]
+      d_green_41_95_start_a: 128
+      d_green_41_95_end_a: 255
+      d_grey_44_start_a: 0
+      d_grey_44_end_a: 159
+      d_grey_122_start_a: 0
+      d_grey_122_end_a: 95
+    second_sheet:
```

Comment:
What does this signify?

Reply:
these are the hardcoded values taken from program as it is